### PR TITLE
Setup Media Financiers

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Setup media financiers
Remove ethers & ngrx dependancies
Move template ids from env into backend-functions

### CODE REVIEW

File: apps/backend-functions/src/main.ts [hunk 1]
```diff
@@ -245,19 +238,7 @@ export const onOrganizationDeleteEvent = onDocumentDelete(
...
-};
-
-export const relayerDeploy = functions.runWith({ timeoutSeconds: 540 }).https
-  .onCall((data, context) => logErrors(relayerDeployLogic(data, RELAYER_CONFIG)));
-
-export const relayerRegister = functions.runWith({ timeoutSeconds: 540 }).https
-  .onCall((data, context) => logErrors(relayerRegisterENSLogic(data, RELAYER_CONFIG)));
 
-export const relayerSend = functions.https
-  .onCall((data, context) => logErrors(relayerSendLogic(data, RELAYER_CONFIG)));
```
Comment (phpgeekfr) [inline apps/backend-functions/src/main.ts:260]: If you are removing ethersjs completety, there is some env var (RELAYER_CONFIG for example) to remove

File: apps/financiers/financiers-e2e/src/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io"
+}
```
Comment (phpgeekfr) [inline apps/financiers/financiers-e2e/src/fixtures/example.json:4]: Will we need these files in this fixture directory ?

File: apps/financiers/financiers/src/app/dashboard/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<ng-container *ngIf="(hasAcceptedMovies$ | async) else noAcceptedMovies">
```
Comment (phpgeekfr) [inline apps/financiers/financiers/src/app/dashboard/home/home.component.html:1]: this content will be changed?

File: apps/financiers/financiers/src/app/dashboard/tunnel/movie-tunnel.component.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+import { extractMediaFromDocumentBeforeUpdate } from '@blockframes/media/+state/media.model';
+
+// Material
+import { MatDialog } from '@angular/material/dialog';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+// RxJs
+import { switchMap } from 'rxjs/operators';
+import { of } from 'rxjs';
+
```
Comment (phpgeekfr) [inline apps/financiers/financiers/src/app/dashboard/tunnel/movie-tunnel.component.ts:19]: @fritzschoff cleaned this on develop

File: tsconfig.json [hunk 5]
```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "./tsconfig.base.json"
-}
```
Comment (phpgeekfr) [inline tsconfig.json:3]: not needed ?

**Review by phpgeekfr — APPROVED** [review]

File: apps/backend-functions/src/main.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/backend-functions/src/main.ts:260]: We don't want to remove ethers completely. We just want to remove it from the build

File: apps/financiers/financiers-e2e/src/fixtures/example.json (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers-e2e/src/fixtures/example.json:4]: No idea, I'll let Mano take care of that.

File: apps/financiers/financiers/src/app/dashboard/home/home.component.html (hunk 3 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers/src/app/dashboard/home/home.component.html:1]: yes, I'm just setting up the app here. I'll go to every page later.

**Pull request merged**